web/src/routes/backOffice/employees/components/EmployeeForm.svelte

```diff
@@ -214,8 +260,58 @@
 			formErrorMessages.location = "";
 		}
 
+		if (
+			roleValidity !== null &&
+			passwordInput !== null &&
+			confirmPasswordInput !== null
+		) {
+			//  Role Validation.
+			if (roleValidity.valueMissing) {
+				formErrorMessages.role = $t("error.valueMissing");
+			} else {
+				formErrorMessages.role = "";
+			}
+
+			// Password Validation.
+			if (passwordInput.validity.valueMissing) {
+				formErrorMessages.password = $t("error.valueMissing");
+			} else {
+				formErrorMessages.password = "";
+			}
+
+			if (confirmPasswordInput.validity.valueMissing) {
+				formErrorMessages.confirmPassword = $t("error.valueMissing");
+			} else if (passwordInput.value !== confirmPasswordInput.value) {
+				formErrorMessages.confirmPassword = $t(
+					"employees.error.passwordMismatch",
+				);
+			} else {
+				formErrorMessages.confirmPassword = "";
+			}
+		}
+
+		if (createForm && passwordInput !== null && confirmPasswordInput !== null) {
+			return (
+				!formErrorMessages.username &&
+				!formErrorMessages.password &&
+				!formErrorMessages.firstName &&
+				!formErrorMessages.lastName &&
+				!formErrorMessages.role &&
+				!formErrorMessages.dateOfBirth &&
+				!formErrorMessages.phoneNumber &&
+				!formErrorMessages.scheduleStart &&
+				!formErrorMessages.scheduleEnd &&
+				!formErrorMessages.location &&
+				!formErrorMessages.password &&
+				!formErrorMessages.confirmPassword &&
+				passwordInput.value === confirmPasswordInput.value &&
+				!!coordinate
+			);
```

Comment:
Instead of duplicating the return statement, I suggest using a more dynamic approach where an array filled with each condition of the return statement would be looped through to find any `false` value. Using this approach, for the create form case, the conditions that are specific to creating an employee can be pushed onto the array before the array is looped.